migrations/2021_03_02_040500_change_access_tokens_add_id.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Schema\Builder;
+
+return [
+    'up' => function (Builder $schema) {
+        $schema->table('access_tokens', function (Blueprint $table) {
+            $table->unsignedInteger('id')->first();
+        });
+
+        $tokens = $schema->getConnection()->table('access_tokens')
+            ->cursor();
+
+        // Insert initial value for our new primary key
+        // This migration runs after the "add type" migration, this ensures we have a minimal number of tokens remaining
+        foreach ($tokens as $i => $token) {
```

Comment:
How long did this take in your environment? I don't know if there's a way to set all these with one query, but even with clearing out the vast majority of tokens, this could be thousands of queries on larger environments